pkg/kv-cache/indexer.go

```diff
@@ -44,7 +44,7 @@ func NewDefaultConfig() *Config {
 	return &Config{
 		PrefixStoreConfig:    prefixstore.DefaultConfig(),
 		TokenProcessorConfig: DefaultTokenProcessorConfig(),
-		KVBlockIndexerConfig: kvblock.DefaultIndexerConfig(),
+		KVBlockIndexConfig:   kvblock.DefaultIndexConfig(),
```

Comment:
Q: is it possible to make the "nil/default" object a sane default so you don't have to initialize it explicitly?